Bigtable/src/Instance.php

```diff
@@ -0,0 +1,331 @@
+            'name' => $this->name,
+            'projectId' => $this->projectId
+        ]);
+
+        return $this->info;
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * $operation = $instance->create();
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     *
+     * @param array $options [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $name.
+     *     @type int $instanceType **Defaults to** `0`.
+     *     @type array $labels For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     *     @type array $clusters [] {
+     *           array {
+     *                 string $clusterId
+     *                 string $locationId
+     *                 int $nodeCount **Defaults to** `1`.
+     *                 int $storageType **Defaults to** `0`.
+     *          }
+     * }
+     * @return LongRunningOperation<Instance>
+     * @codingStandardsIgnoreEnd
+     */
+    public function create(array $options = [])
+    {
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $instanceId = InstanceAdminClient::parseName($this->name)['instance'];
+        $displayName = isset($options['displayName']) ? $options['displayName'] : $instanceId;
+        $type = isset($options['instanceType']) ? $options['instanceType'] : 0;
+        $labels = isset($options['labels']) ? $options['labels'] : [];
+        $clusters = isset($options['clusters']) ? $options['clusters'] : [];
+
+        $operation = $this->connection->createInstance([
+            'parent' => $projectName,
+            'instanceId' => $instanceId,
+            'instance' => [
+                'displayName' => $displayName,
+                'type' => $type,
+                'labels' => $labels
+            ],
+            'clusters' => $this->clustersArray($clusters)
+        ]);
+        return $this->resumeOperation($operation['name'], $operation);
+    }
+
+    /**
+     * @param array $args [] {
+     *      array {
+     *             string $clusterId
+     *             string $locationId
+     *             int $nodeCount **Defaults to** `1`.
+     *             int $storageType **Defaults to** `0`.
+     *      }
+     * }
+     * @return array
+    */
+    private function clustersArray($args)
+    {
+        $clusters = [];
+        foreach ($args as $value) {
+            $id = $value['clusterId'];
+            $clusters[$id] = [
+                'location' => InstanceAdminClient::locationName($this->projectId, $value['locationId']),
+                'serveNodes' => isset($value['nodeCount']) ? $value['nodeCount'] : 1,
```

Comment:
Please use the same term as the service.  So in this case, please use `serverNodes`.  Also, since this is optional if the instance type is `DEV`, please don't set a default.